package.json

```diff
@@ -36,6 +37,8 @@
     "babel-polyfill": "6.16.0",
     "babel-preset-es2015": "6.18.0",
     "copy-webpack-plugin": "4.0.0",
+    "express": "4.14.0",
+    "express-session": "^1.15.5",
```

Comment:
Lets lock down this version as we do other dependencies: `1.15.5`

Reply:
Done